
uint64 timestamp		# time since system start (microseconds)

uint64 timestamp_sample		# the timestamp of the raw data (microseconds)

float32[3] xyz			# Bias corrected acceleration (including gravity) in body axis (in m/s^2)
